stencils/Subject.stencil

```diff
@@ -81,16 +81,3 @@ class MappedSubject<
     }
 }
 
-class MappedWithDefaultSubject<
```

Comment:
It was idea to do as Observable, so class above (MappedSubject should be without default value). If nobody uses that (nothing is broken), I think it's fine